pkg/util/cgroup/manager/v1/fs_linux.go

```diff
@@ -231,6 +232,42 @@ func (m *manager) GetMemory(absCgroupPath string) (*common.MemoryStats, error) {
 	return memoryStats, nil
 }
 
+func (m *manager) GetNumaMemory(absCgroupPath string) (map[int]*common.MemoryNumaMetrics, error) {
```

Comment:
will it be better if we move all the `get` function to metric-fetcher as a new implementation?

Reply:
it's hard to say, these `get` functions only cover a little part of all metric-fetcher scenario so if we do it there will be a lot of effort. Anyway, we shouldn't do it in this PR, maybe we can discuss it later then create a new PR to do that.